src/index.html

```diff
@@ -11,7 +11,156 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+
+      <fieldset class="input">
+        <legend>Personal information</legend>
+        <div class="formfield block">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname:"
+            id="surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="formfield block">
+          <label for="name">Name:</label>
+          <input
+        type="text"
+        name="name:"
+        id="name"
+        autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            name="name:"
            id="name"
            autocomplete="off"
          >
```
Форматування слід виправити скрізь, а не лише там, де я минулого разу вказав. То було як приклад

Reply:
Посилання на тести не працює. Потірбно передеплоїти. Как это сделать? я пробую, все равно ссылка не работает, иду четко по инструкции https://github.com/mate-academy/layout_task-guideline#how-to-solve-the-layout-tasks-on-github, если тесты запускаю показывает что все в порядке, ошибок 0.